.github/workflows/ci.yml

```diff
@@ -36,6 +36,7 @@ jobs:
 
       - name: Install Mercurial
         run: pip install mercurial
+        if: ${{ matrix.os != 'windows-2022' }}
```

Comment:
Using `startsWith` might be more versatile.

Reply:
We don't need python and pip anymore on Windows, do we?

Reply:
Good point. Applied!